vivarium/core/serialize.py

```diff
@@ -155,24 +70,7 @@ def deserialize(self, data: Any) -> List[Any]:
 class DictSerializer(Serializer):  # pylint: disable=abstract-method
```

Comment:
How does this work without any defined codecs? Is it because PyMongo handles dicts for us?

Reply:
Yup, PyMongo does not allow you to create custom serializers for built-in types. The dictionary, identity, and sequence serializers are really only used for deserialization. I've renamed them as such in 5394975.